transactions/remove-approved-nodes/2024/oct-8/README.md

```diff
@@ -1,9 +1,11 @@
 # Remove nodes from the Approved Node Operators ID List
 
-> Oct 1st, 2024
+> Oct 8th, 2024
```

Comment:
Block Daemon has approved the FLIP - see [comment on forum post](https://forum.flow.com/t/flip-gov-291-remove-two-nodes-from-the-approved-list-of-nodes/6599/2?u=vishal)